platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -550,9 +550,9 @@ with the `MGLSymbolStyleLayer.text` property.
 
  Key | Value Type
  --- | ---
- `MGLFontNamesAttribute` | `NSArray<NSString *>*`
- `MGLFontSizeAttribute` | `NSNumber`
- `MGLFontColorAttribute` | `UIColor` or `NSColor` on macos
+ `MGLFontNamesAttribute` | `NSExpression`
+ `MGLFontScaleAttribute` | `NSExpression`
+ `MGLFontColorAttribute` | `NSExpression`
```

Comment:
It’s correct that each of these attributes is set to an expression, but what does the expression contain? We should still tell the developer that, for instance, `MGLFontColorAttribute` is set to an NSExpression containing a UIColor (on iOS) or an NSColor (on macOS).

(Also, this might be a good opportunity to generate separate “Predicates and Expressions” guides for iOS and macOS using an .ejs file, similar to “Information for Style Authors”.)

Reply:
Following the separation of docs on https://github.com/mapbox/mapbox-gl-native/issues/14210